erpnext-openmrs/src/main/java/com/ozonehis/eip/erpnext/openmrs/Constants.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright © 2024, Ozone HIS <info@ozone-his.com>
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.ozonehis.eip.erpnext.openmrs;
+
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor
```

Comment:
Since classes like this shouldn't be instantiatable:

```suggestion
@NoArgsConstructor(access = AccessLevel.PRIVATE)
```